libcontainer/intelrdt/intelrdt.go

```diff
@@ -50,6 +57,7 @@ import (
  * "resource control" filesystem.
  *
  * Intel RDT "resource control" filesystem hierarchy:
+ *
```

Comment:
Line 49:
```
In Linux 4.14 kernel or newer
```
Intel RDT monitoring features (CMT and MBM) were introduced in v4.14 upstream kernel.

Line 56~57 should be removed for the changes in line 49
```
- * CAT and MBA features are introduced in Linux 4.10 and 4.12 kernel via
- * "resource control" filesystem.
```